liblangutil/CharStream.cpp

```diff
@@ -73,6 +73,12 @@ char CharStream::rollback(size_t _amount)
 	return get();
 }
 
+char CharStream::seek(size_t _location)
+{
+	m_position = _location;
```

Comment:
Should perform bounds check.

Reply:
- maybe make rollback call this or vice-versa?

Reply:
> Should perform bounds check.

Good! Will do in next iteration. 

> maybe make rollback call this or vice-versa?

I don't understand what this means. (I am a C++ novice).  If you would like to suggest specific code, I'll make sure it happens.